packages/react/src/components/dropzone.tsx

```diff
@@ -46,18 +46,90 @@ export type UploadDropzoneProps<TRouter extends FileRouter> =
     };
     className?: string;
     config?: {
+      preview?: boolean;
       mode?: "auto" | "manual";
     };
   };
 
+const FilesPreview = ({
+  files,
+  updateFiles,
+}: {
+  files: File[];
+  updateFiles: React.Dispatch<React.SetStateAction<File[]>>;
+}) => {
+  return (
+    <div className="flex flex-row flex-wrap items-center justify-center gap-2">
+      {files.map((file) => (
+        <div
+          key={file.name}
+          className="group relative flex h-24 w-24 flex-col items-center justify-center rounded-lg  p-1 ring-1 ring-gray-200 sm:h-32 sm:w-32"
```

Comment:
maybe `h-full w-auto rounded-sm` 
`<IconX className="absolute top-5 left-5 group-hover:text-red-600 text-red-400">`
enlarge the preview and take up more space will look better?